vm/systemSmartContracts/auction.go

```diff
@@ -1165,7 +1212,158 @@ func (s *stakingAuctionSC) claim(args *vmcommon.ContractCallInput) vmcommon.Retu
 	return vmcommon.Ok
 }
 
-func (s *stakingAuctionSC) calculateNodePrice(bids []AuctionData) (*big.Int, error) {
+func (s *stakingAuctionSC) unStakeTokens(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if !s.flagStakingV2.IsSet() {
+		//unStakeTokens function will become available after enabling staking v2
+		s.eei.AddReturnMessage("invalid method to call")
```

Comment:
add the error message as the commented line

Reply:
This will break the compatibility if someone will send a transaction that calls this method now or before we release this code.

Reply:
Maybe "unStakeTokens function is disabled" like you did for claim above

Reply:
No, will break the compatibility. It has to remain like this